pom.xml

```diff
@@ -21,7 +21,7 @@
     <description>TIS Revalidation Recommendation project</description>
 
     <properties>
-        <java.version>11</java.version>
+        <java.version>17</java.version>
```

Comment:
This is causing the build to fail, not sure if it's an easy fix or maybe something we can't just casually add in (need its own ticket?)